aikau/src/main/resources/alfresco/lists/AlfSortablePaginatedList.js

```diff
@@ -240,7 +240,7 @@ define(["dojo/_base/declare",
                }
                else
                {
-                  this.loadData();
+                  this.onReloadData();
```

Comment:
The onReloadData function was added in a previous release and the AlfSearchList is overriding it. It's typically called as a result of publishing the reload data topic, however we need to call it directly here to get the overridden behaviour in AlfSearchList when sorting.